spec/models/miq_policy_spec.rb

```diff
@@ -307,4 +307,25 @@
       )
     end
   end
+
+  context '.validates' do
+    it 'validates towhat and mode' do
+      expect(described_class.create!(:towhat      => "Host",
```

Comment:
You don't need to save the record to test this, you can 
```ruby
expect(FactoryGirl.build(:miq_policy, :towhat => "Host", :mode => "compliance").valid?).to be true
```

Reply:
Should `mode` even be tested here since it didn't change in this PR?

Reply:
I'm usually not a huge fan of this syntax, but you could also

```ruby
expect(FactoryGirl.build(:miq_policy, :towhat => "Host")).to be_valid
```

Reply:
@bdunne Thank you for the input. I'll have the new test focus on `towhat`

I like the syntax the way it is and if you don't have strong feelings about it I'd like to leave it as is, mostly because it's more consistent with the tests through the spec.

Let me know if you are OK with leaving the test as is.

Thank you. JoeV

Reply:
I guess I don't see the point of writing to the database if we just want to make sure the record is valid.  Also, using `FactoryGirl` makes it much more obvious that you're only testing `:towhat` and not any other validation.

Reply:
I guess the question is: are you testing that it's valid or testing if it raises an exception.  If you're testing validity:  you don't need to create the record, you can do as @bdunne suggests.  If you're testing exception raising, you need to `create!`

It looks like your testing validity so `create!` isn't needed here.  It's minor and I know other code does this but it's fine to mix patterns in the specs until you can convert the rest to the new way.

Reply:
@bdunne and @jrafanie Ah I see the reasoning now. Thank you for explaining. Will do.

JoeV

Reply:
yeah, I prefer expecting the specific thing (validity) rather than the side effect of rails raising an error when calling `create!` on an invalid instance.